### PR TITLE
Cow cloner

### PR DESCRIPTION
**Pull request by richard-to**
# Changes

- Updates tiled map to include cow cloner layer
- Replaces cow box with cow cloner
- Cow cloner is pushable by player
- Cow cloner random shoots cows every 4 secs (max 5 for now)
- Cows will randomly move (need to improve this logic)
- There is a  bug where the cloner can be pushed through the wall

### CODE REVIEW

File: server/game/sprites/items.js [hunk 1]
```diff
@@ -139,6 +141,79 @@ class Cow extends Ingredient {
...
+
+  setClonerSource(cloner) {
+    this.clonerSource = cloner
+  }
+
+  shootUp() {
+    // Logic for shooting a cow from the cloner
+    // Randomly pick the x/y velocities so the cows fall in different directions
+    this.body.angularVelocity = 500
+    this.setVelocityY(random(-600, -1000))
```
Comment (elderferreiras) [inline server/game/sprites/items.js:175]: Phaser.Math.Between generates a random number between that range. No need to change, but it's nice that Phaser offers some of these helping functions: https://photonstorm.github.io/phaser3-docs/Phaser.Math.html

Comment (richard-to) [reply server/game/sprites/items.js:175]: Ah nice. Didn't know about that. I did see that when I was looking at your super elder code (I was looking at it to see how you did the timed projectiles, lol)

Comment (richard-to) [reply server/game/sprites/items.js]: Updated.

**Comment by elderferreiras** [comment]
This is such a cool change

**Pull request merged**